src/Mobile.Bff.Shopping/Extensions/Extensions.cs

```diff
@@ -2,10 +2,7 @@
 {
     public static void AddApplicationServices(this IHostApplicationBuilder builder)
     {
-        builder.Services
-            .AddReverseProxy()
```

Comment:
This change removes the free-for-all reverse proxy and instead maps only specific URL patterns (see `Program.cs` below), and forwards those as needed.